exercises/practice/hamming/test/hamming_test.dart

```diff
@@ -47,12 +44,12 @@ void main() {
       expect(() => hamming.distance('ATA', 'AGTG'), throwsA(mustEqual));
     }, skip: true);
 
-    test('disallow left empty strand', () {
-      expect(() => hamming.distance('', 'G'), throwsA(noEmptyStrand));
+    test('disallow empty first strand', () {
+      expect(() => hamming.distance('', 'G'), throwsA(mustEqual));
     }, skip: true);
 
-    test('disallow right empty strand', () {
-      expect(() => hamming.distance('G', ''), throwsA(noEmptyStrand));
```

Comment:
It seems noEmptyStrand is no longer used in this file. Unless I'm missing anything, you should be free to remove it.

Reply:
Verified it's been removed